### PR TITLE
Add Integration Script for Release Builds

### PR DESCRIPTION
**Pull request by jparr721**
Closes #<ISSUE_NUMBER>

### This PR: 
Adds a script in the scripts folder for tracking the weekly release build.

### CODE REVIEW

File: scripts/auto-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Auto-Integration
+
+To use this set up a virtualenv as
+```shell
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
```
Comment (rob-maron) [inline scripts/auto-integration/README.md:8]: Love the instructions

File: scripts/auto-integration/run-integration.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+REPOS = [
+    "HotShot",
+    "hotshot-query-service",
+    "hotshot-events-service",
+    "hotshot-builder-core",
+    "espresso-sequencer",
+]
+
+PORT_RANGE = list(range(24000, 24005))
```
Comment (rob-maron) [inline scripts/auto-integration/run-integration.py:40]: Maybe we can make the port range specified by arg? Not a big deal though

**Review by rob-maron — APPROVED** [review]
Looks good, but since it touches the `native-demo`, what do you think about it being in the sequencer repo?

File: scripts/auto-integration/README.md (hunk 1 above)
Comment (dailinsubjam) [reply scripts/auto-integration/README.md:8]: +1

**Pull request merged**